.github/CONTRIBUTING.md

```diff
@@ -35,6 +35,16 @@ Contributions are welcome! Here's how you can help:
      - Open the [pull-request form](https://github.com/minetest/minetest/pull/new/master).
      - Add a description explaining what you've done (or if it's a work-in-progress - what you need to do).
 
+### Important note about automated GitHub checks
+
+When you submit a pull request, GitHub automatically runs checks on the Minetest Engine combined with your changes. One of these checks is called 'cpp lint / clang format', which checks code formatting and checks for code style errors.
+
+Because formatting for readability requires human judgement, and because an automated formatting check has limited configurability, this check often fails and ClangFormat often makes unsuitable formatting requests which make code readability worse.
```

Comment:
s/often/sometimes

Reply:
Under the current terrible config, it is often

Reply:
Yes, approx 50% is unsuitable requests in my experience.
Even with improved configuration it will still be 'often' as CF cannot be made to ignore code alignment, apparently.
Best to be honest to contributors about this so that they are prepared.